spring-xd-dirt/src/main/java/org/springframework/xd/dirt/rest/StreamsController.java

```diff
@@ -67,7 +68,24 @@ public StreamsController(StreamDeployer streamDeployer) {
 	@ResponseStatus(HttpStatus.OK)
 	public PagedResources<StreamDefinitionResource> list(Pageable pageable,
 			PagedResourcesAssembler<StreamDefinition> assembler) {
-		return listValues(pageable, assembler);
+
+		PagedResources<StreamDefinitionResource> pagedResources = listValues(pageable, assembler);
```

Comment:
We could preferably do the masking of the `definition` at the `StreamDefinitionResourceAssembler` rather than doing it explicitly below.